app/templates/home.html

```diff
@@ -1,11 +1,28 @@
 {% extends "base.html" %}
```

Comment:
Move to `/templates/demo/send_email.py` if it is not integral part of the system